server/src/services/trash.service.ts

```diff
@@ -38,9 +38,29 @@ export class TrashService {
   }
 
   async empty(auth: AuthDto): Promise<void> {
-    const assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) =>
+    let assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) =>
+      this.assetRepository.getByUserId(pagination, auth.user.id, {
+        trashedBefore: DateTime.now().toJSDate(),
+        isArchived: true,
```

Comment:
Use `withArchived` instead without duplicating the loop.

Reply:
duh, simple! Thanks